08_advanced/memory_snapshots/whisper.py

```diff
@@ -0,0 +1,76 @@
+# # Memory snapshots for Whisper Tiny
+#
+# This program runs inference using OpenAI's [`whisper-tiny`](https://huggingface.co/openai/whisper-tiny).
+# We improve cold boot times by loading the memory into CPU and creating
+# a memory snapshot, then moving weights to a GPU during startup.
+
+import modal
+
+image = (
+    modal.Image.debian_slim()
+        .pip_install("transformers", "datasets", "hf_transfer", "torch", "librosa", "soundfile")
```

Comment:
Also these deps should be `==` pinned. Ref _Review of synmon on Examples_ in our Notion.